packages/cli/src/index.ts

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env node
+
+import * as core from "@arethetypeswrong/core";
+import { Option, program } from "commander";
+import chalk from "chalk";
+import { readFile } from "fs/promises";
+import { FetchError } from "node-fetch";
+
+import * as tabular from "./render/index.js";
+import { readConfig } from "./readConfig.js";
+import { problemFlags } from "./problemUtils.js";
+
+export interface Opts {
+  raw?: boolean;
+  packageVersion?: string;
+  fromFile?: boolean;
+  summary?: boolean;
+  emoji?: boolean;
+  vertical?: boolean;
+  flipped?: boolean;
+  color?: boolean;
+  strict?: boolean;
+  quiet?: boolean;
+  configPath?: string;
+  ignore?: string[];
+}
+
+program
+  .addHelpText("before", "ATTW CLI (v0.0.1)\n")
+  .addHelpText("after", "\ncore: v0.0.6, typescript: v5.0.0-dev.20230207")
+  .version("v0.0.1")
+  .name("attw")
+  .description(
+    `${chalk.bold.blue(
+      "Are the Types Wrong?"
+    )} attempts to analyze npm package contents for issues with their TypeScript types,
+particularly ESM-related module resolution issues.`
+  )
+  .argument("<package-name>", "the package to check; by default the name of an NPM package, unless --from-file is set")
+  .option("-f, --from-file", "read from a file instead of the npm registry")
```

Comment:
Am in the midst of doing this - is it ok if I move `parsePackageSpec` into `core`? Otherwise, I'll end up copying it (which is fine - but I feel like it also might fit into `core`? up to you...)